draft-packages/collapsible/docs/SingleCollapsible.stories.tsx

```diff
@@ -52,7 +52,7 @@ const SingleCollapsibleNoPadding = (): JSX.Element => (
     </ListItem>
     <ListItem>
       <Paragraph variant="body">
-        In that case you should use the 'noSectionPadding' prop.
+        In that case you should use the &#39;noSectionPadding&#39; prop.
```

Comment:
Is this due to the `no-escaped-entities` rule? I am unsure if this is the best developer experience having to write them like this?
How necessary is it as well? I'd expect 🤞 we're all using the same charset so there shouldn't be any problems?

Reply:
As discussed with @gyfchong, I have changed them to `&quot;` and `&apos;` :)